backend/src/utils/email_send.rs

```diff
@@ -0,0 +1,166 @@
+use chrono::Utc;
```

Comment:
Please split the functions in this file to several small functions, and add unit tests of them. For example, your can have a signature function with some real-time param as inputs(e.g. date). So that you can use unit tests to ensure that your signature mechanism is right (e.g. provide a fixed date and you will get a fixed corresponding answer).

See function and unit tests in `burrow_valid.rs`.